services/tasks/alert.go

```diff
@@ -364,8 +364,8 @@ func (t *TaskRunner) alertInfos() (string, string) {
 
 	if t.Task.Version != nil {
 		version = *t.Task.Version
-	} else if t.Task.BuildTaskID != nil {
-		version = "build " + strconv.Itoa(*t.Task.BuildTaskID)
+	} else if t.Task.GetIncomingVersion(t.pool.store) != nil {
```

Comment:
`GetIncomingVersion` do database request. Let's call once this method.

Reply:
will make necessary change